app/controllers/PropertyRepresentationController.scala

```diff
@@ -68,4 +71,76 @@ class PropertyRepresentationController @Inject() (
      }
    }
 
+  def appointableToAgent(ownerId: Long,
+                         agentCode: Long,
+                         checkPermission: Option[String],
+                         challengePermission: Option[String],
+                         paginationParams: PaginationParams,
+                         sortfield: Option[String],
+                         sortorder: Option[String],
+                         address: Option[String],
+                         agent: Option[String]) = authenticated { implicit request =>
+
+    groupAccountsConnector.withAgentCode(agentCode.toString) flatMap {
+      case Some(agentGroup) => propertyLinksConnector.appointableToAgent(
+        ownerId = ownerId,
+        agentId = agentGroup.id,
+        checkPermission = checkPermission,
+        challengePermission = challengePermission,
+        params = paginationParams,
+        sortfield = sortfield,
+        sortorder = sortorder,
+        address = address,
+        agent = agent).map(x => Ok(Json.toJson(x)))
+      case None =>
+        Logger.error(s"Agent details lookup failed for agentCode: $agentCode")
+        NotFound
```

Comment:
Not Found like this blews up HttpVerbs sometimes, test this condition on the frontend.